architecture/overview.adoc

```diff
@@ -5,22 +5,68 @@
 :icons:
 :experimental:
 
-OpenShift is a layered system designed to expose underlying Docker and Kubernetes concepts as accurately as possible, with a focus on easy composition of applications by a developer (install Ruby, push code, add MySQL). Unlike 2.x, more flexibility of configuration is exposed after creation in all aspects of the model. The concept of an application as a separate object is being removed in favor of more flexible composition of "services" - allowing two web containers to reuse a DB, or expose a DB directly to the edge of the network.
+OpenShift is a layered system designed to expose underlying
+https://www.docker.com/[Docker] and
+http://kubernetes.io/[Kubernetes] concepts as accurately as possible, with a focus on easy composition
+of applications by a developer (install Ruby, push code, add MySQL). Unlike
+OpenShift 2.x, more flexibility of configuration is exposed after creation
+in all aspects of the model. The concept of an
+link:../whats_new/applications.html[application] as a separate object
+is being removed in favor of more flexible composition of "services", allowing
+two web containers to reuse a database or expose a database directly to the
+edge of the network.
 
 ==== What are those layers?
 
-Docker provides the abstraction for packaging and creating Linux based lightweight containers.  Kubernetes provides the cluster management and orchestrating Docker containers on multiple hosts.  OpenShift adds source code management, builds, and deployments for developers, and managing and promoting images at scale as they flow through your system, application management at scale, and team and user tracking for organizing a large developer organization.
+https://www.docker.com/[Docker] provides the abstraction for packaging and
+creating Linux based lightweight containers.
+http://kubernetes.io/[Kubernetes] provides the cluster management and
+orchestrating Docker containers on multiple hosts. OpenShift adds:
 
+* Source code management, link:/../dev_guide/builds.html[builds], and link:../dev_guide/builds.html[deployments] for developers
+* Managing and promoting images at scale as they flow through your system
+* Application management at scale
+* Team and user tracking for organizing a large developer organization
 
 ==== What is the OpenShift architecture?
 
-OpenShift is a set of microservices that can run on top of (or alongside) a Kubernetes cluster.  Data about the objects is stored in etcd - a reliable clustered key value store.  Those services are broken down by function:
+OpenShift is a set of microservices that can run on top of (or alongside) a
+http://kubernetes.io/[Kubernetes] cluster. Data about the objects is stored in
+etcd - a reliable clustered key value store. Those services are broken down by
```

Comment:
I'd mark it up as `*etcd*` since it's the name of the software, the service, the package, etc.